Jenkinsfile

```diff
@@ -6,29 +6,19 @@
 
 @Library(['private-pipeline-library', 'jenkins-shared']) _
 
-def workDir = "./"
 
 dockerizedBuildPipeline(
-  buildImageId: "${sonatypeDockerRegistryId()}/cdi/golang-1.14:1",
+  pathToDockerfile: 'jenkins/Dockerfile',
   prepare: {
     githubStatusUpdate('pending')
-    runSafely '''
-    curl -O https://releases.hashicorp.com/terraform/0.13.3/terraform_0.13.3_linux_amd64.zip
-    unzip terraform_0.13.3_linux_amd64.zip
-    '''
   },
-  dockerArgs: "-e WORKSPACE:workspace",
   buildAndTest: {
-    dir(workDir) {
-      runSafely '''
-      go get github.com/jstemmer/go-junit-report
-      go mod tidy
-      go mod vendor
-      go test ./... -v 2>&1 -p=1 | go-junit-report > test-results.xml
-      WORKSPACE=workspace make test
```

Comment:
`make test` is doing pretty much the same thing as `go test ./...` No need to run it twice.  Not a fan of Makefiles, there is enough customization in this one that it's probably worth figuring out.